src/Symfony/Component/DependencyInjection/Compiler/AttributeAutoconfigurationPass.php

```diff
@@ -48,6 +48,13 @@ protected function processValue($value, bool $isRoot = false)
                 $configurator($conditionals, $attribute->newInstance(), $reflector);
             }
         }
+        foreach ($reflector->getMethods() as $method) {
```

Comment:
Let's scan only ` ReflectionMethod::IS_PUBLIC`, and let's skip static methods.

Reply:
Done